app/controllers/registrations_controller.rb

```diff
@@ -1,17 +1,18 @@
 class RegistrationsController < Devise::RegistrationsController
-protected
+
+  protected
 
   def sign_up_params
     params.require(:user).permit(:token, :name, :email, :password, :password_confirmation)
   end
 
- def after_sign_up_path_for(resource)
-     @users_count=User.count('id')
-      if Statistic.exists?
-        Statistic.update(registered_users: @users_count) 
-      else
-        Statistic.create(registered_users: @users_count)
-      end
+  def after_sign_up_path_for(resource)
+    @users_count=User.count('id')
+    if Statistic.exists?
+      Statistic.update(registered_users: @users_count) 
+    else
+      Statistic.create(registered_users: @users_count)
+    end
```

Comment:
- Use `Statistic.update_statistic(registered_users: @users_count)`
- In assignments (eg. line 10), leave a space on either of `=`.